PULL_REQUEST_MANAGEMENT.md

```diff
@@ -0,0 +1,14 @@
+# Rules for managing Pull Requests
+
+For the sake of trying to maintain an acceptable number of open but idle PRs, the following rules should be considered:
+- If a PR has at least one approval, it can be merged after 1 week of waiting for additional comments.
+  - If you consider the change trivial enough or it has at least 3 approvals, it may also be merged right away.
```

Comment:
Superfluous space.
Also priority in this sentence is off, I'd say:
> If the change has **at least 3** approvals or you consider it trivial enough, it may be merged right away.

Reply:
Which space is superfluous? If you mean the indentation, I did that on purpose so it becomes a sub-point of the previous one.

Reply:
Sorry, should have clicked "View" first :)